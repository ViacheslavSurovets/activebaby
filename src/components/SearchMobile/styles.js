import styled from 'styled-components';
import { mobile, tablet } from '@utils/media';

export const SearchMobileContent = styled.figure`
  display: none;
  ${ tablet },
  ${ mobile }{
    display: flex;
    justify-content: center;
    align-items: center;
    height: 100%;
    flex-basis: 7rem;
    &:hover{
      div {
        display: flex;
      }
    }
  }
`;