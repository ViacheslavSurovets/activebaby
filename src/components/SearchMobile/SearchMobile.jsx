import React from 'react';
import { SearchIcon, HeaderSearchDropdown } from '@components';
import { SearchMobileContent } from './styles';


const SearchMobile = () => (
  <SearchMobileContent>
    <SearchIcon color='white' />
    <HeaderSearchDropdown />
  </SearchMobileContent>
);

export default SearchMobile;
