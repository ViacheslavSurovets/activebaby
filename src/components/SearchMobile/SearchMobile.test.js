import React from 'react';
import { shallow } from 'enzyme';
import SearchMobile from './SearchMobile';


describe ( 'SearchMobile', () => {

  it ( 'SearchMobile Snapshot', () => {
    expect ( shallow ( <SearchMobile  /> ) ).toMatchSnapshot ();
  } );
} );
