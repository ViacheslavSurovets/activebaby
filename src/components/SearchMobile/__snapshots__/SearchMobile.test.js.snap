// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchMobile SearchMobile Snapshot 1`] = `ShallowWrapper {}`;
